### ISSUE TITLE
Not allowed to access sources for a GPL licensed extension

### ISSUE THREAD

**Issue by nazmifr** [mozilla/addons#409]
### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)
Hi,
I was wandering on the available extensions and wanted to see the code of this one:
[link](https://addons.mozilla.org/en-US/firefox/addon/return-of-the-curly-lips/)

### What happened?
So I scrolled down to "version information" and I saw a "see the source" link, when I did so I had a nice error that told me I wasn't allowed to access that ressource. Even worse, on some other extensions, there is no "see source" link, why ? Doesn't the Mozilla license allow the review of the source code of the extensions, and what about the GNU GPL, does anyone knows more than I do and would be willing to enlighten me?

### What did you expect to happen?
Have access to the source code.

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)
[screenshot of the error](https://lut.im/4TMLukSBHz/6frV0V2H7pdXMKMB.png)

Thanks in advance

**Comment by rctgamer3** [comment]
Dupe of mozilla/addons-server#5135. See also mozilla/addons-server#5431, which landed this week.
tl;dr disabled because of performance issues, will hopefully return soon™

**Comment by nazmifr** [comment]
Oh okay, thanks for the quick response.
